content/build/guide/multi-platform.md

```diff
@@ -21,70 +21,58 @@ $ docker build --target=server --platform=linux/arm/v7 .
-supporting additional features.
+## Enable the containerd image store
 
-## Buildx setup
+{{< tabs >}}
+{{< tab name="Docker Desktop" >}}
 
-Buildx comes pre-bundled with Docker Desktop, and you can invoke this build
-client using the `docker buildx` command. No need for any additional setup. If
-you installed Docker Engine manually, you may need to install the Buildx plugin
-separately. See
-[Install Docker Engine](../../engine/install/index.md) for instructions.
+To enable the containerd image store in Docker Desktop,
+go to **Settings** and select **Use containerd for pulling and storing images**
+in the **General** tab.
 
-Verify that the Buildx client is installed on your system, and that you’re able
-to run it:
+Note that changing the image store means you'll temporarily lose access to
+images and containers in the classic image store.
+Those resources still exist, but to view them, you'll need to
+disable the containerd image store.
 
-```console
-$ docker buildx version
-github.com/docker/buildx v0.10.3 79e156beb11f697f06ac67fa1fb958e4762c0fab
-```
+{{< /tab >}}
+{{< tab name="Docker Engine" >}}
 
-Next, create a builder that uses the `docker-container`. Run the following
-`docker buildx create` command:
+If you're not using Docker Desktop,
+enable the containerd image store by adding the following feature configuration
+to your `/etc/docker/daemon.json` configuration file.
 
-```console
-$ docker buildx create --driver=docker-container --name=container
+```json {hl_lines=3}
+{
+  "features": {
+    "containerd-snapshotters": true
+  }
+}
 ```
 
-This creates a new builder with the name `container`. You can list available
-builders with `docker buildx ls`.
+Restart the daemon after updating the configuration file.
 
 ```console
-$ docker buildx ls
-NAME/NODE           DRIVER/ENDPOINT               STATUS
-container           docker-container
-  container_0       unix:///var/run/docker.sock   inactive
-default *           docker
-  default           default                       running
-desktop-linux       docker
-  desktop-linux     desktop-linux                 running
+$ systemctl restart docker
 ```
 
-The status for the new `container` builder is inactive. That's fine - it's
-because you haven't started using it yet.
+{{< /tab >}}
+{{< /tabs >}}
 
 ## Build using emulation
 
-To run multi-platform builds with Buildx, invoke the `docker buildx build`
-command, and pass it the same arguments as you did to the regular `docker build`
-command before. Only this time, also add:
+To run multi-platform builds, invoke the `docker build` command,
+and pass it the same arguments as you did before.
+Only this time, also add a `--platform` flag specifying multiple architectures.
 
-- `--builder=container` to select the new builder
-- `--platform=linux/amd64,linux/arm/v7,linux/arm64/v8` to build for multiple
-  architectures at once
-
-```console
-$ docker buildx build \
+```console {hl_lines=4}
+$ docker build \
```

Comment:
Attempting this command does not work, can it be changed to a working example?

```
[+] Building 0.0s (0/0)                                                                                                                                                                                                                                          docker:desktop-linux
ERROR: Multi-platform build is not supported for the docker driver.
```

Reply:
@NiklasBr I guess you aren't using the containerd image store.

Reply:
Also that error message looks like it's coming from an older CLI version, if you update you should get a better prompt

Reply:
SBOM and containerd enabled, Rosetta disabled

Reply:
I forgot I had participated in [#371](https://github.com/docker/roadmap/issues/371) which shows that the above command does not work globally.

Please disregard my comment, it appears this is for future functionality and not something supposed to work today.